lib/src/platform_util/platform_util.dart

```diff
@@ -0,0 +1,26 @@
+library dart_dev.src.platform_util.platform_util;
+
+import 'dart:async';
+
+import 'package:dart_dev/src/platform_util/standard_platform_util.dart';
+
+PlatformUtil platformUtil = new StandardPlatformUtil();
+
+abstract class PlatformUtil {
+  static PlatformUtil retrieve() {
+    if (platformUtil == null) throw new StateError(
+        'dart_dev\'s PlatformUtil instance must not be null.');
+    return platformUtil;
+  }
+
+  /// Generates an HTML report for an LCOV formatted coverage file.
+  // TODO: Future<bool> generateLcovHtml(String lcovPath, String outputPath);
```

Comment:
That was intentional. I want to eventually have the report generation go through here since it's interacting with an executable, and so that it can be mocked. But it will require more than a simple Future<bool> return type because I need to be able to listen to the stdout/stderr streams.